incubator/hnc/pkg/validators/config_test.go

```diff
@@ -0,0 +1,239 @@
+package validators
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	api "github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/api/v1alpha1"
+	. "github.com/onsi/gomega"
+	"k8s.io/api/admission/v1beta1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+func TestDeletingConfigObject(t *testing.T) {
+	t.Run("Delete config object", func(t *testing.T) {
+		g := NewGomegaWithT(t)
+		req := admission.Request{
+			AdmissionRequest: v1beta1.AdmissionRequest{
+				Operation: v1beta1.Delete,
+				Name:      "config",
+			}}
+		config := &Config{}
+
+		got := config.Handle(context.Background(), req)
+
+		logResult(t, got.AdmissionResponse.Result)
+		g.Expect(got.AdmissionResponse.Allowed).Should(BeFalse())
```

Comment:
It seems allowed to delete config object according to the doc but here it's expected to be not allowed?

Reply:
It complies with the validator's behavior you have above though.

Reply:
Thanks for pointing it out! I updated the doc. It should deny deleting the `config` object.